src/caselawclient/models/search_results.py

```diff
@@ -0,0 +1,52 @@
+from typing import List
+
+from lxml import etree
+
+
+class SearchResults:
```

Comment:
Same

public ui: https://github.com/nationalarchives/ds-caselaw-public-ui/blob/6c966bba962e3a8efdd951029b68fa9cb576b3e5/judgments/models.py#L128

editor ui: https://github.com/nationalarchives/ds-caselaw-editor-ui/blob/fd8f1e9100a8a60f081eecb79836f44688e70c94/judgments/models/__init__.py#L143

but converter from using django-xml to just use lxml

Still need to do a followup PR where I move in SearchResult and SearchMatch - bit more complicated though and not entirely necessary for the search_results test so left for separate PR